### PR TITLE
Testsecondary

### PR DESCRIPTION
**Pull request by WilliamPyke**

### CODE REVIEW

File: counters/count.py [hunk 1]
```diff
@@ -6,3 +6,6 @@ def sub(x, y):
 
 def mul(x, y):
     return x * y
+
+def div(x, y):
```
Comment (aidanpinard) [inline counters/count.py:10]: Change variable names to be more descriptive

File: counters/count.py [hunk 2]
```diff
@@ -6,3 +6,6 @@ def sub(x, y):
 
 def mul(x, y):
```
Comment (aidanpinard) [inline counters/count.py:7]: Make full length name

**Review by aidanpinard — CHANGES_REQUESTED** [review]
fix comments

File: counters/count.py (hunk 2 above)
Comment (WilliamPyke) [reply counters/count.py]: Fixed in 857c2f3

File: counters/count.py (hunk 1 above)
Comment (WilliamPyke) [reply counters/count.py]: Fixed in 857c2f3

**Review by aidanpinard — APPROVED** [review]
LGTM

**Pull request merged**